src/Abp.Web.SignalR/RealTime/OnlineClientInfoProvider.cs

```diff
@@ -0,0 +1,45 @@
+using System;
+using Abp.Auditing;
+using Abp.Runtime.Session;
+using Microsoft.AspNet.SignalR.Hubs;
+
+namespace Abp.RealTime
+{
+    public class OnlineClientInfoProvider : AbpServiceBase, IOnlineClientInfoProvider
```

Comment:
I don't think it needs to inherit from `AbpServiceBase`. Currently it just needs `IAbpSession`.

Reply:
In the 0d7540e9615f2f74f01cb780f7b90acc24a28d31 I removed unwanted `AbpServiceBase` inheritance! Thanks!